genrl/core/actor_critic.py

```diff
@@ -77,12 +186,30 @@ def __init__(
         self.action_scale = kwargs["action_scale"] if "action_scale" in kwargs else 1
         self.action_bias = kwargs["action_bias"] if "action_bias" in kwargs else 0
 
+    def get_params(self):
+        actor_params = self.actor.parameters()
+        critic_params = list(self.critic1.parameters()) + list(
+            self.critic2.parameters()
+        )
```

Comment:
Its probably a bad assumption that there will only be two critics given that the class  name is `MlpSingleActorMultiCritic`

Reply:
I had initially used a multi critic version using a list of critics but that blocks backprop and it doesn't train. If you can, see if you can add some other way.

Reply:
I'd say avoid it. This is definitely not the correct way to go about defining multiple critics. We're providing virtually no aid to the user and just making it an extra burden. This should be resolved soon based on changes in the core.

Reply:
Okay, so I'll just rename these to MlpSingleActorTwoCritic then